### DISCUSSION TITLE
Is there a way to search for a photo by its original file name?

### DISCUSSION THREAD

**Discussion by veritasecops** [photoprism/photoprism#3492] [Q&A]
Can it be done?

**Comment by lastzero** [comment]
Yes, see https://docs.photoprism.app/user-guide/search/filters/

**Reply by mamema** [reply]
doesn't working on my side for unkown reasons.
A picture with the name "22885974375_c914ac464e_o" is found in the "Original" area in photoprism, but not found via search string original:"22885974375_c914ac464e_o" in the search area.
Modifying the search string with * doesn't help, even original:"22*" doesn't list the picture

**Reply by lastzero** [reply]
The `original` filter is for imported files only. Simply use `filename` otherwise.

**Comment by mamema** [comment]
filename also doesn't show the picture. 
I've tried to "hunt" the picture with different approaches:
- use tags, labels, dive into the directory via directory menu, nowhere found....

the picture only shows up in the "originals" area. So the pic "is there". I just can't search for it.
- i've reindexed the folder, where the picture is stored, because photoprism does not show all files within this folder (dunno why).
now the picture was shown in the folder view and could be found. But the filename shown in the properties of the picture was including folder names. So a search filename:"*picturename*" was needed....
So this is kinda solved, but the question remains, why was this picture not indexed in the last full index run and which other pictures are missing without notice.....

**Comment by gconklin** [comment]
I had a kind of similar problem not finding some pictures after indexing. Look and see if it is in the Search->Review section
![image](https://github.com/photoprism/photoprism/assets/23655/be76cc08-1845-4063-84c5-1a178f7c2d4e)
If it is, click the check mark under the image.
